ess/src/API/EMBC.ESS.Shared.Contracts/Submissions/EvacuationFiles.cs

```diff
@@ -137,8 +137,8 @@ public class Note
         public string Id { get; set; }
         public NoteType Type { get; set; }
         public string Content { get; set; }
-        public DateTime AddedOn { get; set; }
-        public DateTime ModifiedOn { get; set; }
+        public DateTime CreatedOn { get; set; }
```

Comment:
are you trying to create conflicts for me? :P I'm working on this, if this is not required for this story, can you please undo these changes?